### PR TITLE
This, Proto, stretch

### PR DESCRIPTION
**Pull request by robertjallen**

### CODE REVIEW

File: assignments/prototypes.js [hunk 1]
```diff
@@ -15,13 +15,36 @@
...
 */
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+}
+
+// as a prototype so other objects can use it outside of GameObj
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game`;
```
Comment (AislynnEdmiston) [inline assignments/prototypes.js:26]: Great job using temprate litterals

File: assignments/this.js [hunk 2]
```diff
@@ -13,14 +13,64 @@
 
 // code example for Window Binding
```
Comment (AislynnEdmiston) [inline assignments/this.js:14]: What is an example of window binding?

**Review by AislynnEdmiston — CHANGES_REQUESTED** [review]

File: assignments/prototypes.js (hunk 1 above)
Comment (robertjallen) [reply assignments/prototypes.js:26]: thanx

File: assignments/this.js [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
 
-// console.log(this);
+console.log(this);
```
Comment (AislynnEdmiston) [inline assignments/this.js:16]: Good jod. Sorry must have over looked this comment

**Pull request merged**